### PR TITLE
feat(styles): update badge colors

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/styles/src/variables/components/_badge.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // Design System custom variables
 $badge-height: button.$btn-height-rg;
 $badge-gap: spacing.$size-regular;
-$badge-border-color: color.$gray-20;
+$badge-border-color: var(--post-gray-60);
```
Comment (oliverschuerch) [inline packages/styles/src/variables/components/_badge.scss:15]:
```suggestion
$badge-border-color: color.$gray-60;
```

File: packages/styles/src/variables/components/_badge.scss [hunk 2]
```diff
@@ -38,7 +39,7 @@ $badge-check-input-bg-color: color.$white;
 // Bootstrap variables
 $badge-font-size: type.$font-size-small;
 $badge-font-weight: type.$font-weight-normal;
-$badge-color: color.$gray-60;
+$badge-color: var(--post-gray-80);
```
Comment (oliverschuerch) [inline packages/styles/src/variables/components/_badge.scss:42]:
```suggestion
$badge-color: color.$gray-80;
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/styles/src/variables/components/_badge.scss (hunk 1 above)
Comment (alizedebray) [reply packages/styles/src/variables/components/_badge.scss:15]: I am using CSS variables to make the badge display correctly on backgrounds.

File: packages/styles/src/variables/components/_badge.scss (hunk 2 above)
Comment (alizedebray) [reply packages/styles/src/variables/components/_badge.scss:42]: Same here.

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**